javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,17 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname, useFormalName = false) {
+  if (firstname && surname) {
```

Comment:
Remember to write the Javascript names in camelCase. (Even though I know the homework wrote it incorrectly!)

`getFullName`
`firstName`

Reply:
Changed them.